extensions/libraries/redcore/loader/loader.php

```diff
@@ -24,55 +24,77 @@ class RLoader extends JLoader
 	 * @var    array
 	 * @since  11.1
 	 */
-	protected static $classes = array();
+	protected static $rClasses = array();
 
 	/**
 	 * Container for already imported library paths.
 	 *
 	 * @var    array
 	 * @since  11.1
 	 */
-	protected static $imported = array();
+	protected static $rImported = array();
 
 	/**
 	 * Container for registered library class prefixes and path lookups.
 	 *
 	 * @var    array
 	 * @since  12.1
 	 */
-	protected static $prefixes = array();
+	protected static $rPrefixes = array();
 
 	/**
 	 * Holds proxy classes and the class names the proxy.
 	 *
 	 * @var    array
 	 * @since  3.2
 	 */
-	protected static $classAliases = array();
+	protected static $rClassAliases = array();
 
 	/**
 	 * Holds the inverse lookup for proxy classes and the class names the proxy.
 	 *
 	 * @var    array
 	 * @since  3.4
 	 */
-	protected static $classAliasesInverse = array();
+	protected static $rClassAliasesInverse = array();
 
 	/**
 	 * Container for namespace => path map.
 	 *
 	 * @var    array
 	 * @since  12.3
 	 */
-	protected static $namespaces = array('psr0' => array(), 'psr4' => array());
+	protected static $rNamespaces = array('psr0' => array(), 'psr4' => array());
 
 	/**
 	 * Holds a reference for all deprecated aliases (mainly for use by a logging platform).
 	 *
 	 * @var    array
 	 * @since  3.6.3
 	 */
-	protected static $deprecatedAliases = array();
+	protected static $rDeprecatedAliases = array();
+
+	/**
+	 * Get static property
+	 *
+	 * @param   string  $name  Name of property
+	 *
+	 * @return mixed
+	 */
+	protected static function &getStatic($name)
+	{
+		// Try to use parent property if exists
+		if (property_exists(get_called_class(), $name))
+		{
+			$variable = &static::$$name;
+		}
+		else
+		{
+			$variable = &static::${'r' . ucfirst($name)};
```

Comment:
Shouldnt r be R?

Reply:
I don't think because of the first letter of a variable.
As I know we do not use first uppercase letter for object variables